{{cookiecutter.github_repository}}/{{cookiecutter.main_module}}/graphql/backends.py

```diff
@@ -0,0 +1,22 @@
+from django.contrib.auth import get_user_model
```

Comment:
I believe we don't need this, and everything can be handled by the middleware itself. i.e. figuring out if the token is valid and attaching the authenticated user to the `context`.

Reply:
Makes sense. Updated in [PR](https://github.com/Fueled/django-init/pull/469)